mlos_core/mlos_core/spaces/adapters/llamatune.py

```diff
@@ -403,19 +410,13 @@ def _special_param_value_scaler(
 
         # Check if input value corresponds to some special value
         perc_sum = 0.0
-        ret: float
         for special_value, biasing_perc in special_values_list:
             perc_sum += biasing_perc
             if input_value < perc_sum:
-                ret = param._inverse_transform(special_value)  # pylint: disable=protected-access
-                return ret
+                return float(param.to_vector(special_value))
 
         # Scale input value uniformly to non-special values
-        # pylint: disable=protected-access
-        ret = param._inverse_transform(
-            param._transform_scalar((input_value - perc_sum) / (1 - perc_sum))
-        )
-        return ret
+        return float(param.to_vector(param.to_vector((input_value - perc_sum) / (1 - perc_sum))))
```

Comment:
This doesn't look right.

Reply:
good catch! that was an artifact of an automatic rename